### DISCUSSION TITLE
Script a required field

### DISCUSSION THREAD

**Discussion by hagen31** [sadieblue/agilityblue.workspace-scripts#32] [General]
Is there a way to script into a required field? On the Contacts Object the email address is required and we are using that object for multiple purposes and do not always know the email address. I would like to script the results when that field is null; however, as it is required, I have not been able to do so.

**Comment by bmward80** [comment]
Hi Austin,

You cannot get around system fields that are required on any object. You can ignore field requirements for custom fields via API/Scripting, because those aren't columns directly on the database table for the object, but that's not the case for system fields like "EmailAddress" on the contact object.

Brandon

**Reply by hagen31** [reply]
Hello Brandon,

That is what I thought. I'll have to think of a streamline manual way to update this field when it's not known.
